functions/mailing/mailchimp.js

```diff
@@ -1,38 +1,53 @@
-const axios = require("axios");
 const functions = require('firebase-functions');
+const mailchimp = require("@mailchimp/mailchimp_marketing");
 
 const mailchimpConfiguration = {
   authKey: functions.config().subscribe.mailchimp_api_key,
-  endpoint: functions.config().subscribe.mailchimp_subscription_endpoint,
+  server: functions.config().subscribe.mailchimp_server,
+  user: functions.config().subscribe.mailchimp_user_name,
 };
 
+mailchimp.setConfig({
+  apiKey: mailchimpConfiguration.authKey,
+  server: mailchimpConfiguration.server
+});
+
 const requestConfiguration = {
-  user: {
-    key: mailchimpConfiguration.authKey,
+  auth: {
+    user: mailchimpConfiguration.user,
+    password: mailchimpConfiguration.authKey,
```

Comment:
Reverted from doing an axios request to using the mailchimp API. Authentication is MUCH easier on this end.